src/librados/librados_asio.h

```diff
@@ -117,7 +160,8 @@ auto async_read(ExecutionContext& ctx, IoCtx& io, const std::string& oid,
   return boost::asio::async_initiate<CompletionToken, Signature>(
       [] (auto handler, auto ex, IoCtx& io, const std::string& oid,
           size_t len, uint64_t off) {
-        auto p = Op::create(ex, std::move(handler));
+        constexpr bool is_read = true;
```

Comment:
Is there a significance to these constexpr bool variables or it's just for explicitness (as opposed to passing `true` to `Op::create()` directly)?

Reply:
i just think it improves readability when you give names to function arguments at the call site